packages/builder/src/pages/[application]/frontend/[page]/[screen]/_layout.svelte

```diff
@@ -14,7 +14,9 @@
     if ($leftover) {
       // Get the correct screen children.
       const screenChildren = $store.pages[$params.page]._screens.find(
-        screen => screen.props._instanceName === $params.screen
+        screen => 
```

Comment:
I think we should actually be using the screen Id here, instead of the name... but this fixes a bug for now